pyEpiabm/pyEpiabm/tests/test_unit/test_intervention/test_abstract_intervention.py

```diff
@@ -0,0 +1,40 @@
+import unittest
+
+import pyEpiabm as pe
+from pyEpiabm.intervention import AbstractIntervention
+
+
+class TestAbstractIntervention(unittest.TestCase):
+    """Test the 'AbstractIntervention' class.
+    """
```

Comment:
```suggestion
    """Test the 'AbstractIntervention' class.

    """
```

For small changes like this, you may find it easier to commit the change directly on github. You can also add it to a batch commit (where you commit all of the small changes like this in one commit) using the drop-down button below this comment